api/user/author/model.py

```diff
@@ -1,18 +1,22 @@
+from dataclasses import dataclass
 from datetime import datetime
 
+from flask_login import UserMixin
 from flask_sqlalchemy import SQLAlchemy
 from sqlalchemy import event
 
 from api import db
 
 
-class Author(db.Model):
+@dataclass
+class Author(UserMixin, db.Model):
     id: int = db.Column(db.Integer, primary_key=True)
```

Comment:
Does this auto generate?

Reply:
Do you mean the id? then yes

Reply:
We can move it to UUID as discussed today later.

Reply:
Fine for now. we can bring it in #62